Configuration/TCA/Overrides/static_currencies.php

```diff
@@ -2,8 +2,6 @@
 
 declare(strict_types=1);
 
-defined('TYPO3_MODE') || die();
```

Comment:
As far as I can see, the TCA overrides should be protected with `defined('TYPO3') or die();` instead the old `defined('TYPO3_MODE') or die();` https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/11.0/Deprecation-92947-DeprecateTYPO3_MODEAndTYPO3_REQUESTTYPEConstants.html#typo3-mode-usage-as-global-script-file-security-gate